plugins/QuickScreenshot/main.lua

```diff
@@ -70,8 +71,10 @@ function go()
   -- Different operating systems have different
   -- screenshot utilities, so we check the OS here
   if operatingSystem == "Windows" then
+    app.msgbox("Windows not supported yet.", {})
```

Comment:
You need one button, otherwise you cannot close the msgbox.
```suggestion
    app.msgbox("Windows not supported yet.", {[1] = "OK"})
```